tests/store/handlers/test_createhandler.py

```diff
@@ -111,19 +114,57 @@ def test_add_two_versions_of_bundle(populated_store: Store, second_bundle_data):
     """Test to add two versions of the same bundle."""
     store: Store = populated_store
     # GIVEN a populated store and some modified bundle data
-    assert store._get_query(table=Bundle).count() > 0
+    starting_bundle_count: int = store._get_query(table=Bundle).count()
+    starting_version_count: int = store._get_query(table=Version).count()
+    starting_file_count: int = store._get_query(table=File).count()
 
     # WHEN adding the modified bundle to the database
     new_bundle_obj = store.add_bundle(second_bundle_data)[0]
     store.session.add(new_bundle_obj)
     store.session.commit()
 
-    # THEN there should still be one bundle
-    assert store._get_query(table=Bundle).count() == 1
-    # THEN there should be two versions
-    assert store._get_query(table=Version).count() == 2
-    # THEN tere should be all four files
-    assert store._get_query(table=File).count() == 4
+    # THEN there should still be the same number of bundles
+    assert store._get_query(table=Bundle).count() == starting_bundle_count
+    # THEN there should be one more version
+    assert store._get_query(table=Version).count() == starting_version_count + 1
+    # THEN there should be two more files
+    assert store._get_query(table=File).count() == starting_file_count + 2
```

Comment:
Maybe this test could be a bite more explicit and test that also the "right" File and Version are added?

Reply:
The `add_bundle` function  is tested in another test that does exactly what you want 😊